other/alarms.php

```diff
@@ -1,9 +1,13 @@
 <?php
-
-
 include("../common_functions_v2.php");
+
+// https://github.com/jdorn/sql-formatter/tree/master
+// https://raw.githubusercontent.com/jdorn/sql-formatter/master/lib/SqlFormatter.php
```

Comment:
In lieu of proper requirements, maybe add a version or commit where it is known to work

Reply:
that would be https://github.com/jdorn/sql-formatter/commit/ffecdad6ca3f6235f941e960bc9d290a20054586

Reply:
This is a very dead project, I doubt this will give rise to problems :)